cmd/ack-generate/command/crossplane.go

```diff
@@ -39,11 +34,9 @@ var crossplaneCmd = &cobra.Command{
 	RunE:  generateCrossplane,
 }
 
-var providerDir string
-
 func init() {
 	crossplaneCmd.PersistentFlags().StringVar(
-		&providerDir, "provider-dir", ".", "the directory of the Crossplane provider",
+		&optOutputPath, "provider-dir", ".", "the directory of the Crossplane provider",
```

Comment:
You can/should actually remove this flag entirely since the existing `--output` flag already covers its use:

https://github.com/aws-controllers-k8s/code-generator/blob/31204bc5278e40cf544135b27871fc67d2dc0118/cmd/ack-generate/command/root.go#L118-L120

Reply:
Done.